README-zh-CN.md

```diff
@@ -55,6 +55,7 @@ console.log(converter('香蕉 蘋果 梨')); // output: banana apple pear
 
 **DOM 操作**
 
+HTML属性`lang='*'`定义了目标。
```

Comment:
Need an extra line after this line

Reply:
Rendering is lightly different. I would recommend keeping it concise. See :

Sentences WITHOUT jump.
```test code```

Sentences WITH jump.

```test code```

Reply:
Yes, the rendering is different, but I prefer the second one.